5_recognition.ipynb

```diff
@@ -430,60 +775,109 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "`lego`は、絶対座標上で約（x, y）=（0.6, 0.0）の座標に置かれているようです。"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "`lego` seems to be placed at about (x, y) = (0.6, 0.0) position on absolute coordinates."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "## 認識結果を用いた制御"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "## Control using recognition results"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "以上の認識器を用いて、HSRにレゴブロックを拾わせてみましょう。\n",
     "\n",
     "前章の関数`move_wholebody_ik`に認識結果のx, yを引数として与え、逆運動学によってHSRを制御します。"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "Let's make HSR pick up the Lego block using the above recognizer.\n",
```

Comment:
If we are referring to a previously known Lego block:

    "Let's make the HSR pick up the Lego block using the above recognizer.\n",

If we there is no specific Lego block, but certainly a single Lego block:

    "Let's make the HSR pick up a Lego block using the above recognizer.\n",

Reply:
This part is difficult.

We can only detect "light blue" Lego block. In this sense, may be "the".

But we can detect any Lego block by adjusting hue threshold, this case "a".

Reply:
Perhaps "the light blue Lego block" would work well here:

   "Let's make the HSR pick up the light blue Lego block using the above recognizer.\n",

I see what you mean about "a", but that would only be for the case where you specifically state that they could pick up another block of a different color by changing the hue to be detected.